Assets/Mapzen/PolylineBuilder.cs

```diff
@@ -26,50 +28,82 @@ public PolylineBuilder(MeshData outputMeshData, Options options)
             polygonOptions.MaxHeight = options.MaxHeight;
 
             polygonBuilder = new PolygonBuilder(outputMeshData, polygonOptions);
+            polyline = new List<Vector2>();
         }
 
         PolygonBuilder polygonBuilder;
-
+        List<Vector2> polyline;
         Options options;
 
         Vector2 lastPoint;
         int pointsInLineString;
```

Comment:
Looks like `lastPoint` and `pointsInLineString` are vestigial, we can remove them.